controller/src/main/java/io/pravega/controller/store/stream/StoreException.java

```diff
@@ -23,33 +24,38 @@
      * Enum to describe the type of exception.
      */
     public enum Type {
-        NODE_EXISTS,
-        NODE_NOT_FOUND,
-        NODE_NOT_EMPTY,
+        DATA_EXISTS,
+        DATA_NOT_FOUND,
+        DATA_NOT_EMPTY,
```

Comment:
The NotEmpty is derived from the equivalent name given to the zookeeper exception for node which contains children. Been thinking of a few names but not convincing.